src/System/Boot.tsx

```diff
@@ -63,6 +65,18 @@ export const Boot: React.FC<
     ...context,
   }
 
+  const unleashConfig = {
+    url: `${sd.UNLEASH_API}frontend/`,
+    clientKey: sd.UNLEASH_CLIENT_KEY,
+    refreshInterval: 15,
```

Comment:
A bit confusing that this value is taken in seconds whereas the node library config value with the same name takes milliseconds. Any reason to depart from the library default of 30s as the refreshInterval?

Reply:
Good call. Have added in [90f2431](https://github.com/artsy/force/pull/15263/commits/90f243108dd075e48bc9ba0a0b4c41c1abafda6d) and set to `30` seconds. No specific reason for `15` other than following documented initialization [example](https://docs.getunleash.io/reference/sdks/react#initialize-the-client).